ebos/ecltransmissibility_impl.hh

```diff
@@ -786,7 +788,10 @@ updateFromEclState_(bool global)
     for (auto it = trans.begin(); it != trans.end(); ++it, ++key, ++perform)
     {
         if(*perform)
-            fp->apply_tran(*key, *it);
+            if(grid_.maxLevel()>1) {
+                OPM_THROW(std::invalid_argument, "Calculations on TRANX/TRANY/TRANZ arrays are not support with LGRS, yet.");
+            }
+        fp->apply_tran(*key, *it);
```

Comment:
There seem to be braces missing. Now we will always call the last line. Should only be done if `*perform` is true

Reply:
Thanks, it's corrected now